### PR TITLE
Sync rotation/position with Object3D

### PR DESCRIPTION
**Pull request by dmarcos**

### CODE REVIEW

File: src/core/a-entity.js [hunk 1]
```diff
@@ -720,13 +722,29 @@ var proto = Object.create(ANode.prototype, {
   getAttribute: {
     value: function (attr) {
       // If component, return component data.
-      var component = this.components[attr];
+      var component;
+      if (attr === 'position') { return this.object3D.position; }
+      if (attr === 'rotation') { return this.getRotation(); }
```
Comment (ngokevin) [inline src/core/a-entity.js]: scale as well

File: src/core/a-entity.js [hunk 2]
```diff
@@ -43,6 +43,8 @@ var proto = Object.create(ANode.prototype, {
       this.object3D.el = this;
       this.object3DMap = {};
       this.parentEl = null;
+      this.rotationEuler = new THREE.Euler();
```
Comment (ngokevin) [inline src/core/a-entity.js]: used?

File: src/core/a-entity.js [hunk 3]
```diff
@@ -720,13 +722,29 @@ var proto = Object.create(ANode.prototype, {
...
     },
     writable: window.debug
   },
 
+  getRotation: {
+    value: function () {
+      var radToDeg = THREE.Math.radToDeg;
+      var rotation = this.object3D.rotation;
+      var rotationObj = this.rotationObj;
+      rotationObj.x = radToDeg(rotation.x);
```
Comment (ngokevin) [inline src/core/a-entity.js]: not too long to have written out, consider:

```js
this.rotationObj.x = THREE.Math.degToRad(this.object3D.rotation.x);
this.rotationObj.y = THREE.Math.degToRad(this.object3D.rotation.y)
this.rotationObj.z = THREE.Math.degToRad(this.object3D.rotation.z)
```

Comment (ngokevin) [reply src/core/a-entity.js]: I also think this is better as a private method outside the prototype:

```
getRotation (rotation, target)
```

**Comment by dmarcos** [comment]
done!

**Pull request merged**